plugins/ScheduledReports/templates/_listReports.twig

```diff
@@ -75,12 +75,19 @@
             </td>
             <td>
                 {# download link #}
-                <a href="{{ linkTo({'module':'API', 'segment': null, 'token_auth':token_auth,
-                            'method':'ScheduledReports.generateReport', 'idReport':report.idreport,
-                            'outputType':downloadOutputType, 'language':language,
-                            'format': (report.format in ['html', 'csv']) ? report.format : false
-                       }) }}"
-                   rel="noreferrer noopener" target="_blank" name="linkDownloadReport" id="{{ report.idreport }}" class="link_but withIcon">
+                <form action="{{ linkTo({ 'module':'API', 'segment': null,
+                    'method':'ScheduledReports.generateReport', 'idReport':report.idreport,
+                    'outputType':downloadOutputType, 'language':language,
+                    'format': (report.format in ['html', 'csv']) ? report.format : false }) }}"
+                      method="POST"
+                      target="_blank"
+                      id="downloadReportForm_{{ report.idreport }}"
+                >
+                    <input type="hidden" name="token_auth" value="{{ token_auth }}">
```

Comment:
same here, in theory it will always be an MD5 and safe. but just to be extra safe be good to use `{{ token_auth|e('html_attr') }}`

Reply:
Have fixed both of these